src/main/java/by/intexsoft/customers/products/Application.java

```diff
@@ -0,0 +1,16 @@
+package by.intexsoft.customers.products;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Application {
+    public static void main(String args[]) {
+        Map<String, Long> mapUnOrdered = new HashMap<>();
```

Comment:
Map<String, Long> mapUnOrdered = new HashMap<>();
        mapUnOrdered.put("a", 101L);
        mapUnOrdered.put("b", 101L);
        mapUnOrdered.put("c", 500L);
        mapUnOrdered.put("d", 9000L);
        mapUnOrdered.put("e", 98L);
        mapUnOrdered.put("f", 98L);

please check on this^ input map and threshold=200;

Reply:
Ok, I'll check

Reply:
Yes, anyways need to change the logic, doesn't work proper